test/cppsuite/src/main/thread_worker.h

```diff
@@ -83,6 +83,15 @@ class thread_worker {
      * needs to be rolled back.
      */
     bool remove(scoped_cursor &cursor, uint64_t collection_id, const std::string &key);
+
+    /*
+     * Generic truncate function.
```

Comment:
Can we clarify the arguments to the function here? We have the collection id that we truncate, then two optional keys. And explain the behaviour if none / one or two keys are passed.